app/serializers/api/v1/participant_serializer.rb

```diff
@@ -9,9 +9,18 @@ class ParticipantSerializer
       include JSONAPI::Serializer::Instrumentation
 
       class << self
+        def induction_record(profile, lead_provider)
```

Comment:
i feel like we've gone backwards. to get to this serializer we have a large query to work out all the correct profiles. now we need to go back again to figure out what the correct induction record should be for this profile to show the correct information